packages/web-frontend/src/components/View/ChartWrapper/ReferenceLabel.js

```diff
@@ -7,14 +7,21 @@
 
 import React from 'react';
 
+function returnTrueIfMax(value) {
+  if (value.includes('MAX') || value.includes('Max') || value.includes('max')) return true;
+  return false;
+}
+
 function ReferenceLabel(props) {
   const { fill, fontSize, value, viewBox } = props;
   const x = viewBox.width / 2 + 30;
-  const y = viewBox.y + 15;
+  const defaultY = viewBox.y + 15;
+  const maxY = viewBox.y - 5;
```

Comment:
maxY tells me this is the highest that the Y could be, not that it is the y value for a y label. Maybe we could just do away with the extra variable? :

`const y = isMaxLabel(value) ? viewBox.y - 5 : viewBox.y + 15;`